README.md

```diff
@@ -8,11 +8,11 @@
 // Search.
 $algolia->search('marty mcfly');
 
-// Create global API keys.
-$algolia->addUserKey(['search'], 300);
+// List indexes.
+$algolia->listIndexes();
```

Comment:
This just made me realized I forgot to rename it 😩 Sorry about that!
We're moving to `listIndices` to be more consistent with the rest of Algolia's ecosystem. I documented it as `listIndices` but never fixed the client.
Do mind updating the name?